client/src/locales/de/search.ts

```diff
@@ -3,7 +3,8 @@ const search = {
   hint: {
     hintHeading: 'Hinweis: ',
     hintContent: 'Starte am besten mit einer möglichst spezifischen Suche, '
-    + 'da die Ergebnisanzeige auf 12 Ergebnisse beschränkt ist. '
+    + 'da die Ergebnisanzeige auf 12 Firmen beschränkt ist. '
+    + 'Es gibt Praktika, die bei der gleichen Firma stattgefunden haben, diese zählen nicht zum Limit dazu. '
```

Comment:
```suggestion
    + 'Praktika, die bei der gleichen Firma stattgefunden haben, zählen nur einfach zum Limit dazu. '
```
Mein Versuch einer gut verständlichen Formulierung 😅